spec/spec_helper.rb

```diff
@@ -1,8 +1,7 @@
 require 'rspec/its'
 require 'pry'
-require_relative '../lib/coach'
-
-Coach.require_matchers!
+require 'coach'
```

Comment:
`coach` is on the load path by this point, so we don't need to explicitly require by path.